server.go

```diff
@@ -1258,13 +1266,18 @@ func (sp *serverPeer) OnGetBlocks(_ *peer.Peer, msg *wire.MsgGetBlocks) {
 
 // OnGetHeaders is invoked when a peer receives a getheaders wire message.
 func (sp *serverPeer) OnGetHeaders(_ *peer.Peer, msg *wire.MsgGetHeaders) {
-	// Ignore getheaders requests if not in sync unless the local best chain
-	// is exactly at the same tip as the requesting peer.
-	locatorHashes := msg.BlockLocatorHashes
+	// Send an empty headers message in the case the local best known chain does
+	// not have the minimum cumulative work value already known to have been
+	// achieved on the network.  This signals to the remote peer that there are
+	// not any interesting headers available without appearing unresponsive.
```

Comment:
consider changing `...that there are not any...` to `...that there are no...`. The former is correct, the later reads better I think.

Reply:
Updated to the suggested variant. I don't have a strong preference either way.